cfgov/jinja2/v1/_includes/organisms/header.html

```diff
@@ -46,6 +46,19 @@
             {% include 'molecules/global-search.html' with context %}
 
             <a class="o-header_logo" href="/">
+            {% if flag_enabled('BCFP_LOGO', request) %}
+                <img class="o-header_logo-img"
```

Comment:
Is it worth wrapping just the src, srcset, and sizes and leaving the tags, class, and alt outside the flag?

Reply:
Well, the `alt` has to change, too... it seemed cleaner to just wrap the whole tag.

Reply:
Oh, right, I've already become blind to the change...carry on.